notify_send.py

```diff
@@ -160,6 +160,10 @@
         '/usr/share/icons/hicolor/32x32/apps/weechat.png',
         'Path to an icon to be shown in notifications.'
     ),
+    'desktop-entry': (
+        'weechat',
```

Comment:
As I am unable to test this, I would like to ask: Should the default value be `'weechat'` or `''`? Will `'weechat'` work for other users who want to use this feature or do they need to change it? If they need to change it to make it work, we should probably leave it empty.